plone/dexterity/filerepresentation.py

```diff
@@ -233,7 +234,7 @@ def __getattr__(self, name):
         """Fall back on parent for certain things, even if we're aq_base'd.
         This makes propertysheet access work.
         """
-        if hasattr(self.__parent__.aq_base, name):
+        if hasattr(aq_base(self.__parent__), name):
```

Comment:
@jensens any specific reason to leave the hasattr instead of changing it to a getattr?